src/main/resources/static/css/styles.css

```diff
@@ -6,3 +6,8 @@
 body {
     background-color: whitesmoke;
 }
+
+.mx-auto {
```

Comment:
https://getbootstrap.com/docs/4.0/utilities/sizing/
use 'w-auto' and 'h-auto' classes instead :)